WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -274,9 +274,18 @@ - (ReaderCoordinator *)readerCoordinator
 
 - (NSArray<UIViewController *> *)tabViewControllers
 {
-    return @[self.mySitesCoordinator.rootViewController,
-             self.readerNavigationController,
-             self.notificationsSplitViewController];
+    NSArray *tabs = @[
+        self.mySitesCoordinator.rootViewController,
+        self.notificationsSplitViewController
+    ];
+    
+    NSMutableArray *mutableTabs = [tabs mutableCopy];
+    
+    if (AppConfiguration.showsReader) {
+        [mutableTabs insertObject:self.readerNavigationController atIndex:1];
+    }
+    
+    return mutableTabs;
```

Comment:
Sorry, another nit. Feel free to ignore it. We can avoid using `mutableCopy` by doing:

```suggestion
    if (AppConfiguration.showsReader) {
        return @[
            self.mySitesCoordinator.rootViewController,
            self.readerNavigationController,
            self.notificationsSplitViewController
        ];
    }

    return @[
        self.mySitesCoordinator.rootViewController,
        self.notificationsSplitViewController
    ];
```

Reply:
Fixed! 5922515